dip-0003.md

```diff
@@ -99,9 +101,9 @@ This signature is only required for external collaterals. If collateralOutpoint
 the fact that the ProRegTx was able to move the collateral acts as an implicit proof of ownership. In this case the
 payloadSig must be empty.
 
-The ProRegTx might have the IP address and port fields set to zero. This will result in the masternode being added to the masternode list in the PoSe-banned state and require the operator to issue a ProUpServTx. This is needed in case the owner wants to delegate the operator role to a hosting service without knowing the IP Address and port in advance.
+The ProRegTx might have the masternode network information field empty. This will result in the masternode being added to the masternode list in the PoSe-banned state and require the operator to issue a ProUpServTx. This is needed in case the owner wants to delegate the operator role to a hosting service without knowing the IP Address and port in advance.
```

Comment:
Nit: only suggesting because this sentence is already being modified. 
```suggestion
The ProRegTx might have an empty masternode network information field. This will result in the masternode being added to the masternode list in the PoSe-banned state and require the operator to issue a ProUpServTx. This is needed in case the owner wants to delegate the operator role to a hosting service without knowing the IP Address and port in advance.
```

Reply:
Committed, resolved in latest push